templates/swift/.vscode/tasks.json

```diff
@@ -0,0 +1,13 @@
+{
+    "version": "2.0.0",
+    "tasks": [
```

Comment:
@itowlson I can't get this particular task to load. VS Code simply has no task for "Swift: Build Wasm" even though I can see this file in the .vscode/tasks.json of the project I create. Did I do something obviously wrong here? Or is there an additional step to getting VS Code to pick up my task mapping?

Reply:
Okay, I realized that this was working correctly the whole time. Tested with `run task`